server/liveblog/embed/templates/embed.html

```diff
@@ -1,64 +1,33 @@
 <!doctype html>
 <html ng-app="liveblog-embed">
     <head>
-        <title>Embed</title>
+        <title>{{ blog.title }}</title>
```

Comment:
And then?? You want that I remove the title ? Sorry but having a page title is a must have and not only for SEO.

About the parent page, we can still use the API to retrieve data directly from the parent page.
But it's weird for a newspaper to update the page title through Liveblog. They have a CMS for that.